addons/website/static/src/core/colibri.js

```diff
@@ -42,8 +42,10 @@ export class Colibri {
             this.updateContent();
         };
         for (let node of nodes) {
-            node.addEventListener(event, handler, options);
-            this.handlers.push([node, event, handler, options]);
+            if (node) {
+                node.addEventListener(event, handler, options);
+                this.handlers.push([node, event, handler, options]);
+            }
```

Comment:
I don't think we want this... nodes is either obtained from `getSelectorAll`, and won't contain missing entries - of it is `[target]` - so my guess is that the calling code is explicitly requesting to add an event listener on null.